sql/core/src/main/scala/org/apache/spark/sql/execution/command/functions.scala

```diff
@@ -75,9 +76,18 @@ case class CreateFunctionCommand(
   override def run(sparkSession: SparkSession): Seq[Row] = {
     val catalog = sparkSession.sessionState.catalog
     val func = CatalogFunction(FunctionIdentifier(functionName, databaseName), className, resources)
+    // throws Exception if it does not find the resource
+    def checkIfResourceExists = {
+      resources.foreach(
+        resource =>
+          if (!Utils.isFileExists(resource.uri, sparkSession.sparkContext.hadoopConfiguration)) {
+            throw new AnalysisException(s" Could not find the resource ${resource.uri}")
+          }
+      )
+    }
     if (isTemp) {
       // We first load resources and then put the builder in the function registry.
-      catalog.loadFunctionResources(resources)
+      checkIfResourceExists
```

Comment:
Previously we used to load the function resources. We don't need to do that any more ? Is it done some place else ? If so, we need to adjust the comment above.